src/main/java/org/apache/commons/lang3/RandomUtils.java

```diff
@@ -118,6 +121,7 @@ public static int nextInt(final int startInclusive, final int endExclusive) {
      * @see #nextInt(int, int)
      * @since 3.5
      */
+    @SuppressWarnings("index:argument.type.incompatible") // 0 < Integer.MAX_VALUE
```

Comment:
Why is this error issued at all? The checker should be able to figure out which of two constants is larger! So this looks like a bug that you should report.